Dockerfile.jsonschema

```diff
@@ -0,0 +1,8 @@
+# 3.18.2
+FROM alpine@sha256:25fad2a32ad1f6f510e528448ae1ec69a28ef81916a004d3629874104f8a7f70
+RUN apk add --update protoc=3.21.12-r2 protobuf-dev=3.21.12-r2 go=1.20.5-r0 git=2.40.1-r0
```

Comment:
NB: I picked these version numbers not because they're the most recent, but because they're consistent with Alpine's `world` version for the Alpine version chosen. More recent versions _could_ probably be made to work, but I figured this was the path of least resistance.